logicrunner/builtin/foundation/helpers.go

```diff
@@ -98,6 +94,23 @@ func IsMigrationDaemonMember(member insolar.Reference) bool {
 	return false
 }
 
+func GetMigrationDaemonByMemberRef(memberRef string) (insolar.Reference, error) {
```

Comment:
Do we really need both GetMigrationDaemonByMemberRef and GetMigrationDaemon in foundation. Its better to keep foundation small

Reply:
Done. Move to migration admin